draft-ietf-capport-architecture.xml

```diff
@@ -538,39 +538,39 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
         </section>
       </section>
       <section anchor="id_evaluating">
-        <name>Evaluating an Identifier</name>
+        <name>Evaluating Types of Identifiers</name>
         <t>
-                 To evaluate whether an identifier is appropriate, one should consider
+                 To evaluate whether a type of identifier is appropriate, one should consider
                  every recommended property from the perspective of interactions among
-                 the components in the architecture. When comparing identifiers, choose
+                 the components in the architecture. When comparing identifier types, choose
                  the one which best satisfies all of the recommended properties. The
                  architecture does not provide an exact measure of how well an identifier
-                 satisfies a given property; care should be taken in performing the
+                 type satisfies a given property; care should be taken in performing the
                  evaluation.
         </t>
       </section>
       <section anchor="id_examples">
-        <name>Examples of an Identifier</name>
+        <name>Example Identifier Types</name>
         <t>
-                 This section provides some examples of identifiers, along with some
-                 evaluation of whether they are good identifiers. The list of identifiers
-                 is not exhaustive. Other identifiers may be used. An important point to note
-                 is that whether the identifiers are good depends heavily on the capabilities
+                 This section provides some example identifier types, along with some
+                 evaluation of whether they are good types. The list of identifier types
+                 is not exhaustive. Other types may be used. An important point to note
+                 is that whether the identifier types are good depends heavily on the capabilities
```

Comment:
I think s/the indentifier types are good/a given identifier type is suitable/, perhaps.

Reply:
Took that suggestion. Thanks.